justfile

```diff
@@ -84,7 +84,9 @@ docs-build: docs-build-dependent
 
 markdownlint:
     docker run \
-    -v $PWD:/workdir ghcr.io/igorshubovych/markdownlint-cli:latest \
+    --rm \
```

Comment:
This removes the docker container upon exit. Without this, there will be a lot of unused, old docker containers.